ir/memory.cpp

```diff
@@ -308,8 +308,10 @@ void Pointer::is_dereferenceable(const expr &bytes0, unsigned align,
 
   cond &= is_block_alive();
 
-  if (iswrite)
+  if (iswrite && !m.state->isInitializationPhase()) {
+    // In initialization phase, read-only blocks' values are being updated.
```

Comment:
is this the right place to put this?
Why don't we just disable all checks in store, since we know they all pass anyway?

Reply:
Updated so store calls Pointer::is_dereferenceable with appropriate iswrite